posthog/hogql/grammar/README.md

```diff
@@ -0,0 +1,49 @@
+## How to generate source code files from grammar
+
+Grammar is located inside `HogQLLexer.g4` and `HogQLParser.g4` files.
+
+To generate source code you need to install locally the `antlr4` binary:
+
+```bash
+brew install antlr
+```
+
+or this piece of art if you're using bash on ubuntu for quick access:
+
+```bash
+export ANTLR_VERSION=4.11.1
+
+sudo apt-get install default-jre
+mkdir antlr
+cd antlr
+curl -o antlr.jar https://www.antlr.org/download/antlr-$ANTLR_VERSION-complete.jar
+export PWD=`pwd`
+echo '#!/bin/bash' > antlr4
+echo "java -jar $PWD/antlr.jar \$*" >> antlr4
+chmod +x antlr4
+export CLASSPATH=".:$PWD/antlr.jar:$CLASSPATH"
+export PATH="$PWD:$PATH"
+```
+
+Then either run
+
+```bash
+pnpm run grammar:build
```

Comment:
Homebrew install ANTLR as `antlr` and not `antlr4` so this doesn't quite work for me

Reply:
Since the Ubuntu instructions require manual creation of an `antlr4` executable, maybe those could just use `antlr` too?

Reply:
Fixed! I guess I had both installed and didn't know what was what.